ricecooker/classes/files.py

```diff
@@ -617,7 +617,7 @@ def download_subtitle(self):
 
 class SubtitleFile(DownloadFile):
     default_ext = file_formats.VTT
-    allowed_formats = [file_formats.VTT]
+    allowed_formats = [file_formats.VTT, file_formats.SRT]
```

Comment:
@elaeon You don't need the `SRT` here, adding a `.srt` file will still pass the [validation logic](https://github.com/learningequality/ricecooker/blob/master/ricecooker/classes/files.py#L637-L639) because .srt is one of the `convertible_formats` for subs, see https://github.com/learningequality/le-utils/blob/master/le_utils/resources/presetlookup.json#L60

You can think of `allowed_formats` as the strict file formats that Studio accepts (after conversions).

Reply:
I did a fetch on my brach and when I merged it this line was with the SRT in allowed formats.